.github/workflows/release.yml

```diff
@@ -0,0 +1,99 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      prerelease:
+        description: "Prerelease"
+        required: true
+        default: false
+        type: boolean
+      draft:
+        description: "Draft"
+        required: true
+        default: false
+        type: boolean
+
+permissions:
+  contents: write
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [ "3.7" ]
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Get latest non-prerelease release
+        id: latestrelease
+        uses: cardinalby/git-get-release-action@v1
+        with:
+          latest: true
+          prerelease: false
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Get next minor version
+        id: semverbump
+        uses: WyriHaximus/github-action-next-semvers@v1.2.1
+        with:
+          version: ${{ steps.latestrelease.outputs.tag_name }}
+
+      - name: Update version in code
+        run: |
+          echo "${{steps.semverbump.outputs.minor}}" > devcycle_python_sdk/VERSION.txt
+
+      - name: Commit version change
+        run: |
+          git config --global user.email "github-tracker-bot@taplytics.com"
+          git config --global user.name "DevCycle Automation"
+          git add ./devcycle_python_sdk/VERSION.txt
+          git commit -m "Release $(cat ./devcycle_python_sdk/VERSION.txt)"
+
+      - name: Push version change
+        run: |
+          git push origin main
+        if: inputs.prerelease != true && inputs.draft != true
+
+      - name: Push version change (prerelease)
+        run: |
+          git checkout -B prerelease
+          git push origin prerelease
+        if: inputs.prerelease == true && inputs.draft != true && false # FIXME
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+          cache: 'pip'
+
+      - name: Install dependencies
+        run: |
+          pip install --upgrade pip
+          pip install twine wheel
+
+      - name: Build and validate package
+        run: |
+          python setup.py sdist bdist_wheel
+          twine check dist/*
+
+      - uses: DevCycleHQ/release-action/gh-release@main
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          prerelease: ${{ github.event.inputs.prerelease }}
+          draft: ${{ github.event.inputs.draft }}
+
+      - name: Upload build package to PyPI
+        run: |
+          twine upload --non-interactive dist/*
+        env:
+          TWINE_USERNAME: __token__
+          TWINE_PASSWORD: ${{ secrets.PYPI_TOKEN }}
+        if: inputs.prerelease != true && inputs.draft != true
+
+      # TODO: upload python release artifacts
```

Comment:
Do you have do do anything else at this point?  Isn't the release pushed to PyPi ?

Reply:
I was just thinking of attaching the same package files to the Github release for completeness. It's not strictly necessary, but you'd be able to download and install from Github without PyPI that way.